### ISSUE TITLE
Reproducing the exact tensorflow_inception_v3_stripped_optimized_quantized.pb file

### ISSUE THREAD

**Issue by amirjamez** [tensorflow/tensorflow#20206]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: 14:04
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: r1.9
- **Python version**:  2.7
- **Bazel version (if compiling from source)**: 0.13
- **GCC/Compiler version (if compiling from source)**: 4.8
- **CUDA/cuDNN version**: 7/6
- **GPU model and memory**: 1060
- **Exact command to reproduce**:

Inspecting the available quantized/frozen model of inception-v3 to be used with HVX (`tensorflow_inception_v3_stripped_optimized_quantized.pb`), I see that the spatial_squeeze and some other operations are removed (stripped) and the input and output layers are 'Mul' and 'Softmax' (as they are not defined in nnlib). I tried to reproduce this graph using a pretrained model of my own, but still, end up with spatial_squeeze and thus, it fails to execute on HVX. Also, my input and output layers are called `input` and `InceptionV3/Predictions/Reshape_1`. 

Could you provide the steps to reproduce the exact `tensorflow_inception_v3_stripped_optimized_quantized.pb` using the `graph_transform`/`Optimize_for_inference`/`Quantize_graph` tools, so others can follow these steps to build their own model for HVX? Thanks, @satok16 @petewarden

**Comment by petewarden** [comment]
Unfortunately the old HVX code path is no longer supported, since we're focused on the NN API's implementation instead. One unappetizing alternative is to use an older version of TensorFlow to generate the correct models, but I don't have great directions on that. Closing this as obsolete.
